.github/workflows/test-openmcyclus.yml

```diff
@@ -28,14 +25,12 @@ jobs:
         run: |
           mamba env update -n openmcyclus-env -f environment.yml
```

Comment:
Honestly, I don't know. For this PR I'm kind of in a mind set its works so don't touch it. The weird thing is that even though the `environment.yml` file includes pytest as a dependency, I still had to install it via pip because it wasn't recognized when the tests are run in the CI. So I really don't understand all of it, but it works.